tests/test_integration.py

```diff
@@ -994,20 +994,26 @@ def test_pex_multi_resolve_2():
             ), "{} was not found in wheel".format(dist_substr)
 
 
+if TYPE_CHECKING:
+    TEST_RESOLVE_FN = Callable[[str, str, str, str, Optional[str]], None]
```

Comment:
All caps type names are jarring. I'm guessing you're trying to distinguish an alias from a "real" type via hungarian - but, by definition this is already all local to the file so I'm not seeing the point.

Reply:
My thinking was it was a constant, so following the convention. But I like PascalCase more here.